content/FAQ.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Frequently Asked Questions
+url: FAQ/
+summary: Frequently asked questions
+showToc: true
+tocopen: true
+---
+
+## I read the docs but still need help!
+
+Come join the [discord](https://discord.gg/PeHR6Tjyvn)! Let the community help you in real-time!
+
+## What is VPupPr
+
+VPupPr is an open-source project that animates 3D objects using the glb or vrm file extension. 
+Most notable for using 3D vtube models as its main usage.
+
+## Platform Supported
+
+Currently supported on windows and common linux distributions. MacOS/OSX is not currently supported,
+feel free to fork the github [repo](https://github.com/virtual-puppet-project/vpuppr) and offer it to
+the community!
+
+## Submitting Issues
+
+[Click here](https://github.com/virtual-puppet-project/vpuppr/issues) to submit issues. Be sure to include
+operating system, app version, logs, and any other information related to the bug.
+
+## Building from Source
+
+VPupPr runs off a custom godot engine. Get the pre-compiled engine [here](https://github.com/you-win/godot/releases).
```

Comment:
> VPupPr runs off a custom godot engine

Should be "off of a custom build of the Godot Engine"

> Get the pre-compiled engine

Should be "Get a precompiled build of the engine ..."